doc/en/user/source/introduction/gettinginvolved.rst

```diff
@@ -1,52 +1,66 @@
 
 Mailing lists
 -------------
 
-GeoServer maintains two email lists: `GeoServer Users <http://lists.sourceforge.net/lists/listinfo/geoserver-users>`_ and `GeoServer Developers <http://lists.sourceforge.net/lists/listinfo/geoserver-devel>`_. These lists are publicly available and are a great resource for those who are new to GeoServer, who need a question answered, or who are interested in contributing code. The Users list is mainly for those who have questions relating to the use of GeoServer, and the Developers list is for more code-specific and roadmap-based discussions. If you see a question asked on these lists that you know the answer to, please respond! 
+GeoServer maintains two email lists:
+
+* `GeoServer Users <http://lists.sourceforge.net/lists/listinfo/geoserver-users>`_
+* `GeoServer Developers <http://lists.sourceforge.net/lists/listinfo/geoserver-devel>`_
+
+The Users list is mainly for those who have questions relating to the use of GeoServer, and the Developers list is for more code-specific and roadmap-based discussions. If you see a question asked on these lists that you know the answer to, please respond!
+
+These lists are publicly available and are a great resource for those who are new to GeoServer, who need a question answered, or who are interested in contributing code. 
 
 IRC
 ---
 
-GeoServer has an IRC channel, `#geoserver <irc://irc.freenode.net/geoserver>`_, on the `Freenode <http://freenode.net>`_ network. GeoServer developers frequent this channel, and so it is a great way to give and receive information in real time. 
+GeoServer has an IRC channel, `#geoserver <irc://irc.freenode.net/geoserver>`_, on the `Freenode <http://freenode.net>`_ network. GeoServer developers frequent this channel, and so it is a great way to give and receive information in real time.
 
 Bug tracking
 ------------
 
-If you have a problem when working with GeoServer, then please let us know through the mailing lists. GeoServer uses `JIRA 
-<http://jira.codehaus.org/browse/GEOS>`_ , a bug tracking website, to manage issue reports; you'll need to `create an account first <https://xircles.codehaus.org/>`_. As GeoServer is open source, everyone is encouraged to fix bugs and submit patches. Even if you are not a core developer, we welcome patches through JIRA, or pull requests to `github <https://www.github.com/geoserver/>`_.
+If you have a problem when working with GeoServer, then please let us know through the mailing lists. GeoServer uses `JIRA <http://jira.codehaus.org/browse/GEOS>`_ , a bug tracking website, to manage issue reports. In order to submit an issue, you'll need to `create an account first <https://xircles.codehaus.org/>`_.
+
```

Comment:
Something similar to http://docs.geotools.org/latest/userguide/welcome/support.html#issue-tracker is a handy reference point.